common/membership/ringpop/factory.go

```diff
@@ -82,8 +84,10 @@ func newFactory(
 	tlsProvider encryption.TLSConfigProvider,
 	dc *dynamicconfig.Collection,
 ) (*factory, error) {
-	if err := membership.ValidateConfig(rpConfig); err != nil {
-		return nil, err
+	if rpConfig.BroadcastAddress != "" {
+		if addr := net.ParseIP(rpConfig.BroadcastAddress); addr == nil {
```

Comment:
Why not as it was in `config.go` file?
```suggestion
	if rpConfig.BroadcastAddress != "" && net.ParseIP(rpConfig.BroadcastAddress) == nil {
```